eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -113,8 +113,8 @@ jobs:
         displayName: 'Build and Package'
         inputs:
           mavenPomFile: pom.xml
-          goals: 'package'
-          options: '$(DefaultOptions) "-DpackageOutputDirectory=$(Build.ArtifactStagingDirectory)" -DskipTests -Dinject-codesnippets -Dgenerate-overview -pl $(ProjectList) -am' # We include template-module so we ensure it always builds in CI
+          goals: 'deploy'
+          options: '$(DefaultOptions) -DskipTests -Dinject-codesnippets -Dgenerate-overview -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -pl $(ProjectList) -am -DaltDeploymentRepository=id::default::file://$(Build.ArtifactStagingDirectory)' # We include template-module so we ensure it always builds in CI
```

Comment:
comment on the end can be deleted as it no longer applies.

Reply:
Can we run deploy on just the main project so we can do something 
 like `file://$(Build.ArtifactStagingDirectory)/$(ServiceDirectory)`? Wouldn't that give us the easy segregation of the artifacts so we don't have to do any crazy filename matching in our release templates?

Reply:
I don't think that would help unfortunately since all projects in scope of the reactor would get put into that directory anyway ... unless I am missing something.

Reply:
As long as the output deployment directory doesn't have any thing other then the assets that are going to be published (i.e. they don't contain dependencies and referenced stuff) then I don't think we have to worry about the version number in the directory. We can turn all the logic in our yml files to always take everything under the name of the artifact and eliminated the need for version wildcards and extra copying.

Reply:
@mitchdenny did we give up on the idea of having each artifact set published to its own folder in the "DevOps artifact"?  We have been seeing too many issues with doing the copying in various release tasks and I'd much prefer we avoid those issues by having the exact set of artifacts for a given package under a folder with that package name. Even if we have to do this copying once in the build stage that will make our lives so much easier in the various release stages that need to be correctly configured to copy the correct files.